lib/scihist_digicoll/task_helpers/subject_creator_adjuster.rb

```diff
@@ -0,0 +1,40 @@
+module ScihistDigicoll
+  module TaskHelpers
+    class SubjectCreatorAdjuster
+      attr_accessor :metadata_map, :keys_to_check_for, :changes, :errors
+      
+      def initialize
+        @changes = []
+        @errors = []
+        @metadata_map = {
+          "Bredig, Georg, 1868-" => "Bredig, Georg, 1868-1944",
+          "Caruso, David J."     => "Caruso, David J., (David Joseph), 1978-"
```

Comment:
if we were going to re-use this, we'd want this to be a variable argument somehow of course. We could cross that bridge when we come to it, or we could already make it a variable argument, pass in a hash from rake task?

Reply:
Yeah- this can be turned into a reusable thing next time around.